internal/config/envconfig/load_test.go

```diff
@@ -0,0 +1,69 @@
+package envconfig_test
+
+import (
+	"os"
+	"testing"
+
+	"github.com/git-town/git-town/v19/internal/config/configdomain"
+	"github.com/git-town/git-town/v19/internal/config/envconfig"
+	"github.com/git-town/git-town/v19/pkg/prelude"
+	"github.com/shoenig/test/must"
+)
+
+func TestLoad(t *testing.T) {
```

Comment:
This is how you disable the `tparallel` linter for this entire test function and all its embedded subtests:

```suggestion
//nolint:tparallel
func TestLoad(t *testing.T) {
```

Reply:
It's not you, it's golangci-lint, a sometimes flimsy third-party framework for running dozens on third-party Go linters concurrently. There are two linters at play here: `tparallel` and `paralleltest`. I have found a combination that doesn't produce warnings on CI. 

Btw, to run the linters on your machine, simply run `make lint`.

Reply:
Forgot to mention, I added the fix to your branch. The tests are green now.